### PR TITLE
Single cell updates

### PR DESCRIPTION
**Pull request by bryce-turner**
Some quick updates for single cell, improves adaptability.

Close branch after merge.

### CODE REVIEW

File: modules/single_cell/starsolo.jst [hunk 1]
```diff
@@ -16,6 +16,7 @@
 {% set temp_dir %}temp/{{ sample.gltype }}/expression_estimates/starSOLO/{{ sample.name }}{% endset %}
 {% set task %}starsolo_count_{{ sample.name }}{% endset %}
 {% set json %}{{ alignment_outdir }}/stats/{{ sample.name }}.starSOLO.bam.Barcodes.stats.json{% endset %}
+{% set chemistry = constants.phoenix.scrna_chemistry_options[sample.assayCode] | default(constants.phoenix.scrna_chemistry_options['unknown']) %}
```
Comment (PedalheadPHX) [inline modules/single_cell/starsolo.jst:19]: For star solo I don't think we can do an unknown, the only way to configure correctly is to know

**Review by PedalheadPHX — COMMENTED** [review]
For starSolo I don't think unknown is an option as you can't rely on the autodetect option like cell ranger has

**Comment by bryce-turner** [comment]
Unknown in this case will result in some defaults, not auto detect.

**Review by PedalheadPHX — APPROVED** [review]
Okay still don't love defaults and for cell ranger we might want to migrate tempe to do autodetect, and then remove starSolo to simplify things

**Pull request merged**